.github/workflows/build-asan.yml

```diff
@@ -0,0 +1,20 @@
+name: Build with Memory Leak Checks
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    env:
+      TEST_MAPSCRIPT_PYTHON: 'true'
+      TEST_MAPSCRIPT_PHP: 'true'
+      RUN_MSAUTOTESTS: 'true'
+      RUN_COVERALLS: 'false'
+      WITH_ASAN: 'true'
```

Comment:
you need to explicitly pass them with the -e option of docker run

Reply:
@rouault - not sure how this showed up in the pull request - this was one of several attempts that was since discarded.